aquadoggo/src/graphql/types/document_fields.rs

```diff
@@ -89,6 +87,8 @@ impl DocumentFields {
             .await
             .expect("Schema should be in store");
 
+        println!("{:?} {}", document, name);
+
         // Get the field this query is concerned with.
         match document.get(name).unwrap() {
```

Comment:
I receive a `thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', aquadoggo/src/graphql/types/document_fields.rs:93:34`, investigating 🤔

Reply:
Found the issue, sometimes the document view id is in the past, but I changed a query to only return the latest, fixed my change in the SQL and now its fine again